test/vagrant-service-manager/service_base_test.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../test_helper'
+require 'vagrant-service-manager/config'
+require 'vagrant-service-manager/plugin_util'
+
+module VagrantPlugins
+  describe ServiceManager::ServiceBase do
+    let(:machine) { fake_machine }
+    let(:instance) { ServiceManager::ServiceBase.new(machine, machine.env) }
+
+    it 'should build the proxy options for command properly' do
```

Comment:
@hferentschik I couldn't write the similar tests as [this - `rhn_register_test.rb`](https://github.com/projectatomic/adb-vagrant-registration/blob/master/test/vagrant-registration/cap/redhat/rhn_register_test.rb#L37-L46) you mentioned.
The reason I found that our code to execute command i.e [`PluginUtil.execute_and_exit_on_fail`](https://github.com/projectatomic/vagrant-service-manager/blob/master/lib/vagrant-service-manager/plugin_util.rb#L93) contains more stuff then just executing command like in case of `rhn_register_test.rb`. 

In our case, the method fails on exit_code being empty string and undefined method `zero?` which I found same in `rhn_register_test.rb` where the exit_code is emptry string.